decidim-meetings/app/presenters/decidim/meetings/registration_presenter.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    #
+    # Decorator for meeting registrations
+    #
+    class RegistrationPresenter < SimpleDelegator
+      delegate :name, :email, to: :user
+
+      def status
+        return I18n.t("attended", scope: "decidim.meetings.models.registration.status") if attendee?
+
+        I18n.t("not_attended", scope: "decidim.meetings.models.registration.status")
+      end
+
+      def status_html_class
+        return "success" if attendee?
+
+        "alert"
+      end
+
+      def attendee?
```

Comment:
maybe we don't need this method ? Maybe we could use an alias?

Reply:
You're right, initially I thought that attendance would depend on more variables. I'm going to remove it